Reason: gcc-10 defaults to -fno-common, which results in errors when
global variables are defined multiple times.
Upstream: no
diff --git a/runtime/backtrace.c b/runtime/backtrace.c
index a3c2c08..0a3ea33 100644
--- a/runtime/backtrace.c
+++ b/runtime/backtrace.c
@@ -28,7 +28,7 @@
 #include "caml/fail.h"
 
 /* The table of debug information fragments */
-struct ext_table caml_debug_info;
+extern struct ext_table caml_debug_info;
 
 CAMLexport int32_t caml_backtrace_active = 0;
 CAMLexport int32_t caml_backtrace_pos = 0;
diff --git a/runtime/startup_nat.c b/runtime/startup_nat.c
index 43b85e3..a5479de 100644
--- a/runtime/startup_nat.c
+++ b/runtime/startup_nat.c
@@ -44,7 +44,7 @@
 #endif
 
 extern int caml_parser_trace;
-CAMLexport header_t caml_atom_table[256];
+CAMLextern header_t caml_atom_table[256];
 char * caml_code_area_start, * caml_code_area_end;
 struct ext_table caml_code_fragments_table;
 
